client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js

```diff
@@ -124,6 +125,10 @@ class HistoricalProgramBars extends React.Component {
                   selected: _.toggle_list(selected, id),
                 });
               }}
+              onToggleClick={() => this.setState({
+                selected: selected.length < all_keys.length ? all_keys : [],
```

Comment:
Most of the legend onClicks have a check to prevent the final item from being toggled off, right? What happens when no items are selected?